app/models/event/course.rb

```diff
@@ -59,6 +59,8 @@
 # A course is a specialised Event that has by default applications,
 # preconditions and may give a qualification after attending it.
 class Event::Course < Event
+  LEADER_ROLES = [Event::Role::Leader].map(&:sti_name)
```

Comment:
Since this is supposed to be overridden in Wagons, I would remove the `map` and do this when reading/using this list.

This would then also fit better with the `Event.role_types` which are also stored as constants.